src/components/Users/UserAvatar.tsx

```diff
@@ -58,6 +60,7 @@ export default function UserAvatar({
         if (xhr.status === 200) {
           await sleep(1000);
           refetchUserData?.();
```

Comment:
now that we are invalidating the query, do we need to call `refetchUserData`?
check what the callees are doing with it and see if we can remove this.